Makefile

```diff
@@ -48,7 +48,13 @@ requirements: ## Install requirements for development
 requirements-test: ## Install requirements for testing
 	${VENV_BIN}/pip install -qr requirements/test.txt --exists-action w
 
-migrate:  ## Apply database migrations
+production-requirements:
+	pip install -r requirements.txt --exists-action w
+
+migrate:
+	python manage.py migrate
+
+migrate-dev:  ## Apply database migrations
 	${VENV_BIN}/python manage.py migrate --no-input
```

Comment:
Does this version not work in production? I'd rather just have one single `migrate` target that works for both if possible.

Reply:
Done